Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.21.6-alpine
+FROM nginxinc/nginx-unprivileged:1.21.6-alpine
```

Comment:
This is an official image, so we should be fine. The image difference from `nginx` is as follows:

- The default NGINX listen port is now 8080 instead of 80.
- The default NGINX user directive in /etc/nginx/nginx.conf has been removed.
- The default NGINX PID has been moved from /var/run/nginx.pid to /tmp/nginx.pid. *Change _temp_path variables to /tmp/.

Reply:
The default port has to be changed because ports below 1024 are [priovileged ports](https://www.w3.org/Daemon/User/Installation/PrivilegedPorts.html#:~:text=Priviliged%20ports,has%20put%20up%20for%20you.) and require root privileges. For running the editor with docker only this results in changing the mapped port only (`-p 80:80` will become `-p 80:8080`). Reminder: need to update the docs about that ;)
For end users running swagger editor e.g. with compose or in kubernetes this will be a breaking change as they need to adapt their ports.